.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,7 @@
+# Thank You
+
+We very much welcome contributions to Dojo 2.
+
+Because we have so many repositories that are part of Dojo 2, we have located our [Contributing Guidelines](https://github.com/dojo/meta/blob/master/CONTRIBUTING.md) in our [Dojo 2 Meta Repository](https://github.com/dojo/meta#readme).
```

Comment:
This is just a stream of consciousness thought, but as we are moving to monorepo, does this copy need to change?

Reply:
Yes. Can you file an issue to update this after the merge?

Reply:
Done: https://github.com/dojo/framework/issues/12